src/app/home-page/widget/filtered-list.ts

```diff
@@ -0,0 +1,31 @@
+import { OnInit } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { UserQuery } from 'app/shared/user/user.query';
+import { DefaultService } from 'app/shared/openapi';
+import { Base } from 'app/shared/base';
+
+/**
+ * Base class for logged in homepage widgets that have a filter list
+ */
+export abstract class FilteredList extends Base implements OnInit {
+  userId$: Observable<number>;
+  protected ngUnsubscribe: Subject<{}> = new Subject();
```

Comment:
Now that you're extending Base, you don't need this (it's already in Base)

Reply:
Oops